extensions/raycast-bundlephobia/src/analyze-package.jsx

```diff
@@ -0,0 +1,108 @@
+import { popToRoot, Detail, showToast, Toast } from "@raycast/api";
+import fetch from "node-fetch";
+import { useState, useEffect } from "react";
+
+export default function AnalyzePackage(props) {
+  const { packageName } = props.arguments;
+
+  const [markdown, setMarkdown] = useState("Loading data from Bundlephobia...");
+
+  function formatBytes(bytes) {
+    if (bytes < 1024) {
+      return bytes + "B";
+    } else if (bytes < 1024 * 1024) {
+      return (bytes / 1024).toFixed(1) + "kB";
+    } else if (bytes < 1024 * 1024 * 1024) {
+      return (bytes / (1024 * 1024)).toFixed(1) + "mB";
+    } else if (bytes < 1024 * 1024 * 1024 * 1024) {
+      return (bytes / (1024 * 1024 * 1024)).toFixed(1) + "gB";
+    } else {
+      return (bytes / (1024 * 1024 * 1024 * 1024)).toFixed(1) + " tB";
+    }
+  }
+
+  useEffect(() => {
```

Comment:
I think too much stuff is happening in this useEffect, I would suggest to extract some related stuff to separate functions, like: fetching data, processing / calulating data, generating markdown